### PR TITLE
14 Registering, Signing in and signing out contacts

### PR DESCRIPTION
**Pull request by leonardk1**
#14
As A Contact of the System 
I Want to register to the business
So That I can Perform my specific roles

### CODE REVIEW

**Comment by leonardk1** [comment]
@godfreym please review

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -122,4 +122,52 @@ ul.alert-dropdown {
...
+/* devise alerts */
+/*flash*/
+.alert-error {
+  background-color: #f2dede;
+  border-color: #eed3d7;
+  color: #b94a48;
+  text-align: left;
+ }
+
+.alert-alert {
```
Comment (godfreym) [inline app/assets/stylesheets/custom.css.scss]: @leonardk1 are you not using bootstrap yet? i see some bootstrap classes

**Comment by leonardk1** [comment]
@godfreym the classes issue has been fixed, please review

**Comment by godfreym** [comment]
@leonardk1 Yep now looks good, :+1:

**Comment by leonardk1** [comment]
@arafatm please review

**Comment by arafatm** [comment]
@leonardk1 You're doing too much in this PR. HArd to review.

What is the "Single Responsibility" for this PR? Break that out and resubmit.

**Comment by leonardk1** [comment]
closed as Too Big
will focus on SR

**Pull request closed without merging**